src/helpers/index.ts

```diff
@@ -54,14 +58,22 @@ export const getBase255Number = (color: string, alpha = false): number => {
 export const getCMYKNumber = (color: string): number => Math.min(PCENT.test(color) ? +(color.replace(PCENT, '$1')) / 100 : +color, 1);
 
 //---Return an ordered string from an array values
-export const getOrderedArrayString = (keys: string[]): string => keys.sort().join('').toUpperCase();
+export const getOrderedArrayString = (keys: string[]): string => {
+    const resultKeys = [...keys];
+    resultKeys.sort();
+    return resultKeys.join('').toUpperCase();
+};
```

Comment:
If the intention is avoiding the array mutation, just spread the variable. It does‘t gain much legibility converting it to a block.

```suggestion
export const getOrderedArrayString = (keys: string[]): string =>
    [...keys].sort().join('').toUpperCase();
```